waku/node/rest/health/handlers.nim

```diff
@@ -0,0 +1,37 @@
+when (NimMajor, NimMinor) < (1, 4):
+  {.push raises: [Defect].}
+else:
+  {.push raises: [].}
+
+import
+  chronicles,
+  json_serialization,
+  presto/route
+import
+  ../../waku_node,
+  ../responses,
+  ../serdes
+
+logScope:
+  topics = "waku node rest health_api"
+
+const ROUTE_HEALTH* = "/health"
+
+const futIsReadyTimout = 5.seconds
+
+proc installHealthApiHandler*(router: var RestRouter, node: WakuNode) =
+
+  router.api(MethodGet, ROUTE_HEALTH) do () -> RestApiResponse:
+
+    let isReadyStateFut = node.isReady()
+    if not await isReadyStateFut.withTimeout(futIsReadyTimout):
+       return RestApiResponse.internalServerError("Health check timed out")   
+
+    var msg = "Node is healthy"
+    var status = Http200
+
+    if not isReadyStateFut.read(): 
+      msg = "Node is not inititialized"
+      status = Http503
+
+    return RestApiResponse.textResponse(msg, status)
```

Comment:
unsure about the best practices of `health` endpoints, but would it make sense to return a json that we can extend in the future? In case we want more granularity on the state of the services. For example, we may want to say that is not healthy, but provide a list on which services are not healthy.

No need to define it right now but allow for future extension. return json instead of textResponse?

Reply:
I can change it but in this case http status holds the information rather. 
For meaningful message, text can be enough. Not sure what information we can structure into json later.
For checking status of different submodules or protocols sounds more like an admin interface than a health. Wdyt?

Reply:
yep indeed. http status code has the information.

i was suggesting to return a json in case we want to add more information in the future. so if for example its failing, you have information such as (example taken from prysm node): so you know which service is not healthy

```
/healthz

*rpc.Service: OK
*prometheus.Service: OK
*p2p.Server: OK
*powchain.Web3Service: OK
*attestation.Service: OK
*operations.Service: OK
*blockchain.ChainService: OK
*sync.Service: ERROR not initially synced

```

no need to do it now, but be prepared to extend it.

Reply:
I was gonna add the same comment - let's use JSON rather than text - it is extensible and easier to process with tools than plain text

Reply:
I'm ok with it, but will do in a separate PR is possible.
Also I think we need to enhance a bit client decoders (not just here) to accept json/text both due error cases returns text type, mixing them can cause exception. It's on my list already, but will do in all rest endpoints where necessary.

Reply:
Unrelated to this issue, obviously, but since we are talking about it - would it make more sense to always return json (even for errors)?